profitbricksapi/datacenter.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+namespace ProfitBricksApi;
+
+/**
+ * Class DataCenter describes the properties of a data center at ProfitBricks.
```

Comment:
Here and for all other data-containers (snapshot, virtualdisk, virtualmachine), I would a comment that these are quite simple data-containers which wrap all available parameters and in a well formulated sentence,
that these here defined values can resist (and further used in the code) even if ProfitBricks changes something.
The only need would be to adapt the simple wrapper her.e